02_Data_processing.r

```diff
@@ -185,7 +186,7 @@ Mollusk_York_result <- cbind(newdat_lowT_York, Mollusk_York_pred$fit)
 # Third order polynomial regression following Müller et al., 2019 and Jautzy et al., 2020
 newdat <- data.frame(Temp = seq(0, 1000, 0.1))
 D47m_poly <- lm(D47 ~ poly(I(10^6 / (Temp + 273.15) ^ 2), 3), data = dat, subset = which(dat$D47_outlier == FALSE))
-D47m_poly_pred <- predict.lm(D47m_poly, newdata = newdat, se.fit = TRUE, interval = "confidence", level = 0.95)
+D47m_poly_pred <- predict.lm(D47m_poly, newdata = reaname(newdat_York, Temp = x), se.fit = TRUE, interval = "confidence", level = 0.95)

```

Comment:
this seems a problem?

Reply:
Yes, good catch!

Reply:
seems like I left this in there, better if you fix it on your end to prevent further merge issues ;-).
The binder now seems able to go through all the code, except that some of my things write files to the `out/` directory, which doesn't exist yet. Perhaps we can add a simple `dir.create("out")` to the script?

Reply:
Perhaps also `dir.create("imgs")` and then ggsave the plots as small (say 700x500 pixels) PNGs for this repo? So people can easily inspect the plots without going to the paper?